### DISCUSSION TITLE
detectors_resnet.py

### DISCUSSION THREAD

**Discussion by zhu011** [open-mmlab/mmdetection#9332] [Q&A] [answered]
### Branch

3.x branch https://mmdetection.readthedocs.io/en/3.x/

### 📚 The doc issue

backbone中的detector_resnet的预训练权重链接在哪里可以找到

**Comment by Czm369** [comment, marked as answer]
It will download automatically during the first training session

**Comment by ZwwWayne** [comment]
It still uses the pre-trained model of resnet.
